rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1033,6 +1038,31 @@ public long getFeePerKb(Object[] args)
         return bridgeSupport.getFeePerKb().getValue();
     }
 
+    public long getLockingCap(Object[] args) {
```

Comment:
Something is wrong. 
The code below suggests the value returned is in BTC (no satoshis)
Then the method should be called getLockingCapInBTC()

Reply:
changed it to use satoshis as it was originally intended